language-server/modules/langserver-core/src/test/resources/completion/expression_context/config/mapping_expr_ctx_config47.json

```diff
@@ -36,7 +36,7 @@
       "detail": "LinkedList",
       "sortText": "R",
       "filterText": "fill",
-      "insertText": "next: {},\nvalue: 0",
+      "insertText": "next: {value: 0, next: },\nvalue: 0",
```

Comment:
Previously we've had `{}` assigned to `next`. But with the current change we don't have anything assigned to it.

Reply:
We have done this by design to avoid SO in recursive cases. For level 2 and beyond this will be the behaviour